appserver/packager/external/jakarta-ee11-shim/pom.xml

```diff
@@ -243,7 +259,12 @@
                                     org.jboss.weld.module;
                                     org.jboss.weld.security;
                                     org.jboss.weld.util;
-                                    org.jboss.weld.xml;version="5.0.99";shim=true
+                                    org.jboss.weld.xml;version="5.0.99";shim=true,
+
+                                    <!-- eclipselink packages required by jersey-media-moxy (downgrade) -->
+                                    org.eclipse.persistence.internal.core.helper;
+                                    org.eclipse.persistence.jaxb;
+                                    org.eclipse.persistence.jaxb.rs;version="4.0.99";shim=true
```

Comment:
The safest approach is to shim everything, but probably enough to get by for now if it's only that single Jersey dependency that's complaining (since we're going to be upgrading Jersey Soon™ anyway).

Like Weld, EclipseLink has _many_ package exports (probably more than it should do - why are they exporting "internal" packages?)